frontend/src/types/AuthTypes.ts

```diff
@@ -28,6 +34,17 @@ export type AuthenticatedDonor = {
   instagramLink?: string;
 } | null;
 
+export type AuthenticatedVolunteer = {
+  id: string;
+  firstName: string;
+  lastName: string;
+  email: string;
+  role: Role.VOLUNTEER;
+  accessToken: string;
+  phoneNumber: string;
+  status: Status;
+} | null;
+
```

Comment:
let's just make it a union type
```suggestion
export type AuthenticatedVolunteer = VolunteerResponse | null;
```